docs/shared/jedi.yaml

```
@@ -0,0 +1,27 @@
+platform:
+  account: me
+  scheduler: slurm
+jedi:
+  execution:
+    batchargs:
+      nodes: 1
+      stdout: path/to/runscript.out
+      walltime: 08:00:00
+    envcmds:
+      - module load some-module
+      - module load jedi-module
+    executable: /path/to/jedi
+    mpiargs:
+      - --export=ALL
+      - --ntasks $SLURM_CPUS_ON_NODE
+    mpicmd: srun
+    configuration_file: 
+      base_file: path/to/config.yaml
+      update_values: {"baz": "qux"}
+    files_to_copy:
+      copy_foo: /path/to/foo
+      copy_bar/baz: /path/to/baz
+    files_to_link:
+      link_foo: /path/to/foo
+      link_bar/baz: /path/to/baz
```

```suggestion
    files_to_copy:
      f1: /path/to/f1
      d/f2: /path/to/f2
    files_to_link:
      f3: /path/to/f3
      d/f4: /path/to/f4
```
I was thinking something more like ^^^, as it I would find it surprising to see the same two files (`foo` and `baz`) bot copied _and_ symlinked into a run directory, even if the copies/links in the run directory don't conflict with one another. I think it would be more likely to see four different files, some of them copied, some of them symlinked. That's more like what we see with components' run directories. This example would represent one file and one symlink in the root of the run directory, and then a different file and a different symlink in the `d/` subdirectory of the run directory.